mainpage.dox

```diff
@@ -253,5 +253,10 @@ In libiio, this can be done with two functions, iio_device_reg_read() and iio_de
 The libiio ABI tries to be both backwards and forwards compatible.
 This means applications compiled against an older version will work fine with a newer dynamically linked library. Applications compiled against a newer version will work fine with an older dynamically linked library so long as they don't access any new features. Applications using new features should ensure the libiio version is compatible by using iio_library_get_version() to avoid undefined behavior.
 
+\example iio_attr.c     Part of the libiio-utilites, iio_attr is a utility for displaying information about local or remote IIO devices. By providing an optional value, iio_attr will attempt to write the new value to the attribute.
+\example iio_info.c Part of the libiio-utilites, iio_info is a utility for displaying information about local or remote IIO devices
```

Comment:
The description is not aligned like the rest and it's missing the dot that ends the sentence.

Reply:
fixed